plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/AbstractTestIcebergConnectorTest.java

```diff
@@ -1836,113 +1836,97 @@ public void testIncorrectIcebergFileSizes()
-        assertQuery("SELECT COUNT(nt.b) FROM test_join_partitioning_normal2 nt JOIN test_join_partitioning3 pt ON nt.b = pt.b", "VALUES 4");
-
-        assertUpdate("INSERT INTO test_join_partitioning4 SELECT * FROM test_join_partitioning_normal2", "VALUES 6");
-        assertQuery("SELECT nt.* FROM test_join_partitioning_normal1 nt JOIN test_join_partitioning4 pt ON nt.a = pt.a", normalValues1);
-        assertQuery("SELECT pt.a, pt.b FROM test_join_partitioning_normal2 nt JOIN test_join_partitioning4 pt ON nt.a = pt.a where nt.a > 0", expectedvalues2);
-
-        // join partitioned table with partitioned table
-        String expectedPartitioningValues1 = "VALUES " +
-                "(1, 'b', TIMESTAMP '2020-09-09 10:10:23.123213'), " +
-                "(2, 'c',  TIMESTAMP '2020-09-09 11:10:33.234234')," +
-                "(3, 'd',  TIMESTAMP '2020-09-10 11:10:43.234234')," +
-                "(4, 'e',  TIMESTAMP '2020-09-10 11:10:53.234234')," +
-                "(5, 'f',  TIMESTAMP '2020-09-10 09:10:33.123355')";
-        String expectedPartitioningValues2 = "VALUES " +
-                "(1, 'b')," +
-                "(2, 'c')," +
-                "(3, 'd')," +
-                "(4, 'e')," +
-                "(5, 'f')";
-        assertQuery("SELECT pt1.* FROM test_join_partitioning1 pt1 JOIN test_join_partitioning2 pt2 ON pt1.a = pt2.a", expectedPartitioningValues1);
-        assertQuery("SELECT pt1.* FROM test_join_partitioning1 pt1 JOIN test_join_partitioning3 pt3 ON pt1.b = pt3.b", expectedPartitioningValues1);
-        assertQuery("SELECT pt1.* FROM test_join_partitioning1 pt1 JOIN test_join_partitioning4 pt4 ON pt1.a = pt4.a", partitioningValues1);
-
-        assertQuery("SELECT pt2.a, pt2.b FROM test_join_partitioning2 pt2 JOIN test_join_partitioning3 pt3 ON pt2.a = pt3.a", expectedPartitioningValues2);
-        assertQuery("SELECT pt2.a, pt2.b FROM test_join_partitioning2 pt2 JOIN test_join_partitioning4 pt4 ON pt2.a = pt4.a", expectedPartitioningValues2);
-        assertQuery("SELECT pt3.a, pt3.b FROM test_join_partitioning3 pt3 JOIN test_join_partitioning4 pt4 ON pt3.a = pt4.a", expectedPartitioningValues2);
-
-        assertQuery("SELECT count(pt1.a) FROM test_join_partitioning1 pt1 JOIN test_join_partitioning2 pt2 ON pt1.a = pt2.a", "VALUES 5");
-        assertQuery("SELECT count(pt4.a) FROM test_join_partitioning1 pt1 JOIN test_join_partitioning4 pt4 ON pt1.a = pt4.a", "VALUES 6");
-        assertQuery("SELECT count(pt4.b) FROM test_join_partitioning1 pt1 JOIN test_join_partitioning4 pt4 ON pt1.a = pt4.a", "VALUES 5");
-
-        dropTable("test_join_partitioning1");
-        dropTable("test_join_partitioning2");
-        dropTable("test_join_partitioning3");
-        dropTable("test_join_partitioning4");
-        dropTable("test_join_partitioning_normal1");
-        dropTable("test_join_partitioning_normal2");
+    public void testPartitionedTableWithDateTransformStatistics()
```

Comment:
I think you did not mean to remove this test you have just added. It seems like you are fixing two separate things. It would be best if those were separate PRs.